api_spec.yaml

```diff
@@ -169,7 +168,7 @@ paths:
         - OAuth2 Recipe
       operationId: verifyOAuth2AccessToken
```

Comment:
you can extract the token info return into an object and reuse that in APIs:
- token post
- client id + access token verify
- client id + refresh token 
- client id + session handle